modules/build-run/pages/sca-user-flows.adoc

```diff
@@ -0,0 +1,81 @@
+= Build and run an Application
+:page-aliases: ROOT:sca-user-flows.adoc
+:experimental: // activate the 'menu' macro
+:description: User flows possible in Application mode
+
+== User flows in Application mode
+
+=== Creating a new project in Studio using Application mode
+
+[WARNING]
+====
+In order to generate the docker image, it's mandatory to have a maven repository working to get all the dependencies, including the latest Application runtime version.
+====
+
+. Run Bonita Studio 2023.2 and create a new project.
```

Comment:
```suggestion
. Run Bonita Studio {bonitaVersion} and create a new project.
```

Reply:
It's better to use Antora Attribute for this value. If you use 2023.2, we will need to update this value on next branch's version...